docs/panels.rst

```diff
@@ -5,18 +5,23 @@ Panels
 Basic panels
```

Comment:
do we need this files at all? It looks like they're just displaying some sphinx design options?

Reply:
Hmm, not sure if it's really needed or not.  All the docs here are to show what different design elements look like with the theme applied. I'm not sure if there is special handling for panels in the theme code or not though.

Reply:
ok lets leave it in for now and see about finding a better place for it later